packages/x-vanilla/package.json

```diff
@@ -23,7 +23,8 @@
     }
   },
   "devDependencies": {
-    "happy-dom": "^5.3.1"
+    "happy-dom": "^5.3.1",
+    "vitest": "^0.16.0"
```

Comment:
I could not execute vitest after `cd`ing to this directory without the bin specified here